franka_description/robots/utils.xacro

```diff
@@ -1,6 +1,62 @@
 <?xml version="1.0"?>
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+  <!-- ============================================================ -->
+  <!-- Macro to add an <interial> tag based on yaml-load properties -->
+  <!--                                                              -->
+  <!-- name:     Name of the robot link (without prefix)            -->
+  <!-- inertial: Dictionary of inertia properties                   -->
+  <!-- ============================================================ -->
+  <xacro:macro name="inertial" params="inertial:=^ name:=^">
+    <xacro:if value="$(arg gazebo)">
+      <xacro:unless value="${inertial.get(name, None)}">${xacro.warning('No inertia properties defined for: ' + name)}</xacro:unless>
```

Comment:
The [recommended way](https://docs.python.org/3/reference/expressions.html#membership-test-operations) to test if a key exists in a dict would be: `<xacro:unless value="${name in inertial}">`

Reply:
Fixed.